src/containers/subreddit-threads.js

```diff
@@ -74,11 +76,13 @@ SubredditThreads.propTypes = {
   }).isRequired,
   subredditThreadCardExpandedThumbnails: PropTypes.instanceOf(Map).isRequired,
   dispatch: PropTypes.func.isRequired,
+  nameOfLastThreadFetched: PropTypes.string,
```

Comment:
This can be `undefined` when the app first loads but `fetchSubredditThreads` handles that by defaulting it to `''`.